Finishlinelab3/finishlinelab3infra.yaml

```diff
@@ -57,8 +79,8 @@ Outputs:
     Value: !GetAtt 
       - CharityInstance
       - AvailabilityZone
-  PublicIP:
-    Description: Public IP address of the newly created EC2 instance
+  PublicDNS:
+    Description: Public DNSName of the newly created EC2 instance
     Value: !GetAtt 
       - CharityInstance
```

Comment:
This should be the name of your webserver

Reply:
done